RELEASE.md

```diff
@@ -54,3 +54,19 @@ git push upstream v1.2.0
 
 ### Post-Steps
 Once the tag has been pushed the release action should run automatically. You can view the progress [here](https://github.com/operator-framework/operator-controller/actions/workflows/release.yaml). When finished, the release should then be available on the [releases page](https://github.com/operator-framework/operator-controller/releases).
+
+
+## Backporting Policy
+
+Mainly critical issue fixes are backported to the most recent minor release.
+Special backport requests can be discussed during the weekly Community meeting or via Slack channel;
+this does not guarantee an exceptional backport will be created.
+
+Occasionally non-critical issue fixes will be backported, either at an approver’s discretion or by request as noted above.
+For information on contacting maintainers and attending meetings, check the community repository.
```

Comment:
nit: maybe make community repository a link as well? alternatively/additionally we could also point to the contribution guide in this repository

Reply:
Good Idea